packages/components/src/lib/Alert.tsx

```diff
@@ -29,10 +29,12 @@ const Alert: FunctionComponent<AlertProps> = ({
     setIsOpen(false);
   };
 
-  return isOpen ? (
+  // Without message check the alert is set to empty but the snackbar
+  // persists for a short period when clearToastMessage is called
+  return isOpen && !!message ? (
```

Comment:
I added this as there were some points where a success message, for the link copy, would quickly be followed by an empty error message due to the `clearToastMessage`. From `message` being a required attribute, the assumption is that `message` will always be available to show within the `Snackbar`. If this assumption is incorrect and an empty `""` `message` is a possibility then I will revert this change.